integrations/jira/.port/resources/blueprints.json

```diff
@@ -11,10 +11,17 @@
           "type": "string",
           "format": "url",
           "description": "URL to the project in Jira"
+        },
+        "totalIssues": {
+          "title": "Total Issues",
+          "type": "number",
+          "description": "The total number of issues in the project"
         }
       }
     },
-    "calculationProperties": {}
+    "mirrorProperties": {},
+    "calculationProperties": {},
+    "relations": {}
   },
   {
     "identifier": "jiraIssue",
```

Comment:
I am thinking maybe we should add a property with sprint, where the description of the property will point the user to the docs?

Reply:
The docs already contains how to add the sprint property. You think we should add it by default?